### PR TITLE
consider quantity for addon price

### PR DESCRIPTION
**Pull request by MauserBitfly**
This PR
- takes the quantity into account for the addon price displays on the pricing pages

### CODE REVIEW

File: frontend/components/pricing/PremiumAddonBox.vue [hunk 1]
```diff
@@ -21,17 +21,17 @@ const quantityForPurchase = ref(1)
...
-    monthly: formatPremiumProductPrice($t, props.addon.price_per_month_eur),
-    monthly_based_on_yearly: formatPremiumProductPrice($t, props.addon.price_per_year_eur / 12),
-    yearly: formatPremiumProductPrice($t, props.addon.price_per_year_eur),
+    main: formatPremiumProductPrice($t, mainPrice * quantity),
+    monthly: formatPremiumProductPrice($t, props.addon.price_per_month_eur * quantity),
+    monthly_based_on_yearly: formatPremiumProductPrice($t, props.addon.price_per_year_eur / 12 * quantity),
+    yearly: formatPremiumProductPrice($t, props.addon.price_per_year_eur * quantity),
     saving: formatPremiumProductPrice($t, savingAmount, savingDigits),
-    perValidator: formatPremiumProductPrice($t, mainPrice / props.addon.extra_dashboard_validators, 5)
+    perValidator: formatPremiumProductPrice($t, mainPrice / props.addon.extra_dashboard_validators * quantity, 5)
```
Comment (D13ce) [inline frontend/components/pricing/PremiumAddonBox.vue:34]: **Bug:** You must not multiply here.

**Review by D13ce — CHANGES_REQUESTED** [review]
Can be considered done, but found a bug. PTAL below :point_down:

File: frontend/components/pricing/PremiumAddonBox.vue (hunk 1 above)
Comment (MauserBitfly) [reply frontend/components/pricing/PremiumAddonBox.vue]: good catch, fixed

**Review by D13ce — APPROVED** [review]
LGTM

**Pull request merged**